tests/integration/cassettes/TestLiveThread_test_init.json

```diff
@@ -0,0 +1,164 @@
+{
+  "http_interactions": [
+    {
+      "recorded_at": "2016-12-06T14:54:15",
+      "request": {
+        "body": {
+          "encoding": "utf-8",
+          "string": "grant_type=client_credentials"
+        },
+        "headers": {
+          "Accept": "*/*",
+          "Accept-Encoding": "identity",
+          "Authorization": "Basic <BASIC_AUTH>",
+          "Connection": "keep-alive",
+          "Content-Length": "29",
+          "Content-Type": "application/x-www-form-urlencoded",
+          "User-Agent": "<USER_AGENT> PRAW/4.0.0 prawcore/0.3.0"
+        },
+        "method": "POST",
+        "uri": "https://www.reddit.com/api/v1/access_token"
+      },
+      "response": {
+        "body": {
+          "encoding": "UTF-8",
+          "string": "{\"access_token\": \"tuIhJlqr1dLVDqCFbVqtWF7tynw\", \"token_type\": \"bearer\", \"expires_in\": 3600, \"scope\": \"*\"}"
```

Comment:
Wow, despite writing the OAuth code twice, I mistakenly assumed that the authorization header was also sent in each request. Oops. Thanks for the correction.

There certainly is reason to try to hide the tokens in such cases. Thanks for the correction.